src/org/purefn/starman/jedis.clj

```diff
@@ -113,11 +113,12 @@
 
 (defn default-config
   "k8s config based on env"
-  []
-  (when (k8s/kubernetes?)
-    (-> (k8s/config-map "redis")
-        keywordize-keys
-        (rename-keys {:host ::host}))))
+  ([name]
+   (-> (k8s/config-map name)
+       keywordize-keys
+       (rename-keys {:host ::host})))
+  ([]
```

Comment:
clj convention is to order arity from least to most.
https://github.com/bbatsov/clojure-style-guide#oneline-short-fn